pyproject.toml

```diff
@@ -191,15 +184,8 @@ legacy_tox_ini = """
         PYVISTA_OFF_SCREEN
         XAUTHORITY
     deps =
-        imageio[ffmpeg]
-        magicgui
-        napari
-        pyqt5
-        pytest
         pytest-cov
         pytest-qt
-        pytest-xvfb ; sys_platform == 'linux'
-        qtpy
```

Comment:
It seems with current tests none of these are required?

Reply:
Doesn't look like it.